mods/d2k/mod.yaml

```diff
@@ -173,7 +173,7 @@ Fonts:
 		Font:./mods/common/FreeSansBold.ttf
 		Size:14
 	Title:
-		Font:d2k:Dune2k.ttf
+		Font:./mods/d2k/Dune2k.ttf
```

Comment:
Note: This temporarily breaks the ability to load d2k from the support directory.  This will be restored by #10703.